src/main/features/response/routes/how-much-paid-claimant.ts

```diff
@@ -0,0 +1,71 @@
+import * as express from 'express'
+import { Paths } from 'response/paths'
+
+import { FormValidator } from 'forms/validation/formValidator'
+import { Form } from 'forms/form'
+
+import { ResponseType } from 'response/form/models/responseType'
+import { ErrorHandling } from 'common/errorHandling'
+import { User } from 'idam/user'
+import { GuardFactory } from 'response/guards/guardFactory'
+import { DraftService } from 'services/draftService'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Claim } from 'claims/models/claim'
+import { Draft } from '@hmcts/draft-store-client'
+import { HowMuchPaidClaimant, HowMuchPaidClaimantOption } from 'response/form/models/howMuchPaidClaimant'
+
+function isRequestAllowed (res: express.Response): boolean {
+  const draft: Draft<ResponseDraft> = res.locals.responseDraft
+
+  return draft.document.response !== undefined
```

Comment:
It looks like using `isResponseRejectedFullyWithDispute` would remove duplication and make it more readable.